packages/web-components/fast-element/src/components/controller.ts

```diff
@@ -186,7 +216,13 @@ export class Controller extends PropertyChangeNotifier {
      * Removes styles from this element. Providing an HTMLStyleElement will detach the element instance from the shadowRoot.
      * @param styles - the styles to remove.
      */
-    public removeStyles(styles: ElementStyles | HTMLStyleElement): void {
+    public removeStyles(
+        styles: ElementStyles | HTMLStyleElement | null | undefined
```

Comment:
What is the purpose of allowing `null` and `undefined` here and in `addStyles`?

Reply:
There were two places where the caller of this API had to check null before calling the API, so I thought it might be better to just enable the API to be resilient to that scenario.